lib/uniffi-zcash/src/test_support.rs

```diff
@@ -0,0 +1,156 @@
+use std::collections::HashMap;
+use std::fs::read_to_string;
+// use crate::ZcashError;
+extern crate serde_json;
+
+type OutputsMap = HashMap<String, Vec<u8>>;
+type ArgsMap = HashMap<String, Vec<String>>;
+
+pub struct ZcashTestSupport {
+    outputs: OutputsMap,
+    args: ArgsMap,
+}
+
+impl ZcashTestSupport {
+    pub fn from_csv_file() -> Self {
+        let mut outputs: OutputsMap = HashMap::new();
+        let mut args: ArgsMap = HashMap::new();
+
+        read_to_string("tests/test_data.csv")
```

Comment:
this is the incriminated csv - only Kotlin is OK with this path, the others are not